docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -150,7 +150,7 @@ The format used is SWC ( [http://www.neuronland.org/NLMorphologyConverter/Morpho
 
 It is not required that the soma is located at 0,0,0 in the SWC file, but in cases where the morphology has a soma, the soma will be re-centered to 0,0,0 upon loading into the circuit.  Node translations will then be applied to this recentered morphology. This behavior can be overridden by the optional reserved attribute "recenter" for nodes and node_types.  See "Representing networks of neurons" for more details.
 
-It is recommended, but not required, that morphologies in a network have a standardized orientation, so that their orientation vectors in the network can be inferred from node rotation angles.  For circuits for which this is true, the circuit producer can declare it with the "standardized_morphology_orientation" entry in the circuit_config.json, so downstream users can safely assume it.  See "Tying it all together - the circuit_config file" for more details.
```

Comment:
I removed the second sentence because it is not formalized what "standardized_morphology_orientation" means.